docs/content/content-management/shortcodes.md

```diff
@@ -300,6 +300,36 @@ Using the preceding `tweet` example, the following simulates the displayed exper
 
 {{< tweet 877500564405444608 >}}
 
+### `telegram`
+
+To embed a post from public Telegram channel you need just its URL:
```

Comment:
This also works with public groups, not only channels.

Reply:
Thanks! Updated documentation to reflect this.